### ISSUE TITLE
Lib status and documentation?

### ISSUE THREAD

**Issue by markerikson** [MARTAflex/redux-capsule#1]
Hi.  This library looks potentially interesting, but also looks like it's a work-in-progress and doesn't have any documentation.  Can you clarify what it does, and what the library's status is?

**Comment by cdxOo** [comment]
Hi, I am quite impressed that someone actually found this. It is however a work in progress that we (as a company) are using in our product. I have currently no spare time to write the documentation for this, and i also highly discourage using it at this point in time since the api is currently completely unstable.

However: it is inteded to solve the counter pair problem with redux and react. In this scenario you have written a counter component, and now you want to display this component multiple times. 

There are other projects out there solving this problem, but I wasnt quite satisfied with them in the context of our project, so we started our own.
Theese projects are (as of my knowledge):
* [https://github.com/DataDog/redux-doghouse](https://github.com/DataDog/redux-doghouse)
* [https://github.com/salsita/prism](https://github.com/salsita/prism)
* [https://github.com/HansDP/local-react-redux](https://github.com/HansDP/local-react-redux)

You can find more information about the problem itself there.

I hope I could help you a little.

**Comment by markerikson** [comment]
Yep, I'm very familiar with that problem and some of the other implementations, which is part of why I asked.  FYI, I'm a Redux maintainer, and maintain a list of Redux-related addons at https://github.com/markerikson/redux-ecosystem-links .  If you do get this library to a point where you consider it usable and with some documentation, please submit a PR to my list!

**Comment by cdxOo** [comment]
Thank you, I'll sure do!
